src/main/java/com/bandwidth/sqs/consumer/acknowledger/MessagePublisher.java

```diff
@@ -0,0 +1,9 @@
+package com.bandwidth.sqs.consumer.acknowledger;
+
+import java.time.Duration;
+
+import io.reactivex.Completable;
+
+public interface MessagePublisher<T> {
```

Comment:
Not yet, it has a "Client" but that doesn't know how to publish an arbitrary `T` type